src/mixins/mixin-general.js

```diff
@@ -5,7 +5,7 @@ import { openURL } from 'quasar'
 import { version } from '../../package.json'
 import { Platform } from 'quasar'
 import { LocalStorage } from 'quasar'
-import { countryCodes_rev } from 'app/firebase-functions/shared/src/country-constants.js'
+import { countryCodes, countryCodes_rev } from 'app/firebase-functions/shared/src/country-constants.js'
```

Comment:
Thanks for picking up after me, whatever necessitated this must have been my fault 😳

Reply:
Not your fault, it was me who imported the first line, and only noticed that we have both arrays available in the file, which is awesome!